web/client/components/TOC/fragments/settings/WMSCacheOptions.jsx

```diff
@@ -282,7 +285,7 @@ function WMSCacheOptions({
                         bsStyle={(!supportFormatCache || !selectedTileGridId)
                             ? 'danger'
                             : 'success'}
-                        title={<Message msgId="layerProperties.tileGridInfoChecksTitle" />}
+                        title={<Message msgId="layerProperties.notSupportedSelectedFormatCache" />}
```

Comment:
shouldn't this be similar to the bsStyle rule? 

```suggestion
                        title={<Message msgId={(!supportFormatCache || !selectedTileGridId) ? "layerProperties.notSupportedSelectedFormatCache" : "layerProperties.tileGridInfoChecksTitle"}/>}
```

Reply:
@MV88 I think I edited this by mistake, I made revert for this.